packages/api/src/external/carequality/document/create-outbound-document-retrieval-req.ts

```diff
@@ -72,6 +73,7 @@ export function createOutboundDocumentRetrievalReqs({
       const request: OutboundDocumentRetrievalReq[] = docRefChunks.map(chunk => {
         return {
           ...baseRequest,
+          subRequestId: uuidv4(),
```

Comment:
the idea here is that when we break the request up into 50-100 chunks, we dont have viz on each chunk. This will help with that

Reply:
Also, NABD, but I'd use `nanoid()` here so:
- we minimize growing file name length too much
- jumps to the eye this is a sub req ID